File: cpp/fizzbuzz1.cpp
#include <iostream>

int main() {
  int i;
  for (i = 1; i <= 100; i++) {
    if (i % 15 == 0)
      std::cout << "Fizzbuzz ";
    else if (i % 5 == 0)
      std::cout << "Buzz ";
    else if (i % 3 == 0)
      std::cout << "Fizz ";
    else
      std::cout << i << " ";
  }
  return 0;
}

File: python/fizzbuzz1.py
#!/usr/bin/python
print 'Fizz-Buzz\n'
for i in range(1,100,1):
  if i%3 == 0:
      if i%5== 0:
          print 'FizzBuzz'
      else:
          print 'Fizz'
  elif i%5 == 0:
      print'Buzz'
  else:
      print i
